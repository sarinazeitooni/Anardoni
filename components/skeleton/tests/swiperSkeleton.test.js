import React from 'react';
import { shallow } from 'enzyme';
import SwiperSkeleton from "../Skeleton";
describe("swiper skeleton component", () => {
    it("should render div as props length", () => {
        const skeleton = shallow(<SwiperSkeleton />);
        expect(skeleton.length).toBe(1);
        // expect(skeleton.find("div").length).toBe(10);
        expect(skeleton).toMatchSnapshot();
    });
});