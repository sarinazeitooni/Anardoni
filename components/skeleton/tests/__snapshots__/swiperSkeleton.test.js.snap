// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`swiper skeleton component should render div as props length 1`] = `ShallowWrapper {}`;
